### DISCUSSION TITLE
PWA install requirements

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#221] [Feature Requests]
Hey, @DustinBrett? have you linked the manifest file to the main page yet?
The reason why I'm asking is because the manifest needs to be linked to the main page to work

**Comment by DustinBrett** [comment]
I have not yet done the work for this to be a PWA. I have larger plans for the service worker and at the moment I have not been in a hurry for the feature. But it is for sure planned.

**Reply by CorruptionVR** [reply]
the file you want to use as a PWA manifest is [package.json](https://github.com/DustinBrett/daedalOS/blob/main/package.json) correct?

**Reply by DustinBrett** [reply]
I don't currently have the desire for PWA. If I were to add it, it's a process involving quite a few things to do the way I want. I plan to have the service worker be custom made and to work as a firewall, cache server, web server, etc. When I desired to work on a service worker is when the PWA will be added probably.
